### PR TITLE
Remove downloaded data when running MXNet example

### PR DESCRIPTION
**Pull request by hkvision**

### CODE REVIEW

**Comment by hkvision** [comment]
jenkins passed: http://10.239.47.210:18888/job/ZOO-PR-Validation/3206/

**Comment by hkvision** [comment]
http://10.239.47.210:18888/job/ZOO-PR-Validation/3208/

File: pyzoo/zoo/examples/run-example-test-ray.sh [hunk 1]
```diff
@@ -41,7 +41,13 @@ time4=$((now-start))
 
 echo "#5 Start mxnet lenet example"
 start=$(date "+%s")
+# Remove the downloaded data to avoid failures
+if [ -d "data" ];
```
Comment (qiuxin2012) [inline pyzoo/zoo/examples/run-example-test-ray.sh:45]: It's better to use `-e`.

Comment (qiuxin2012) [reply pyzoo/zoo/examples/run-example-test-ray.sh:45]: #2209

**Pull request merged**